### PR TITLE
Prep for 0.6.0 release

### PR DESCRIPTION
**Pull request by dpatchigolla**
1. Added a safe except when invoke api falls for static analysis
2. Printing error when audit fails
3. Bumped up profiles-rudderstack dependency to 0.19.0

## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/predictions/profiles_mlcorelib/py_native/id_stitcher/audit.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def _run(self, this: WhtMaterial):
             self.run_completed = True
             n_visualisations = cluster_report.counter
         except Exception as e:
+            print(f"An error occurred while running the audit: {e}")
```
Comment (shekhar-rudder) [inline src/predictions/profiles_mlcorelib/py_native/id_stitcher/audit.py:138]: print -> logger

File: src/predictions/setup.py [hunk 2]
```diff
@@ -49,4 +49,7 @@
     ],
     install_requires=install_requires,
     include_package_data=True,
+    package_data={
```
Comment (shekhar-rudder) [inline src/predictions/setup.py:52]: why is this required ?

File: src/predictions/version.py [hunk 3]
```diff
@@ -1 +1 @@
-version = "v0.5.9"
+version = "v0.6.0"
```
Comment (shekhar-rudder) [inline src/predictions/version.py:1]: version file should not be part of this PR. Will be bumped up in the release process

File: src/predictions/setup.py (hunk 2 above)
Comment (dpatchigolla) [reply src/predictions/setup.py:52]: This is for the tutorial. Let me remove this as it's not yet registered.

File: src/predictions/version.py (hunk 3 above)
Comment (dpatchigolla) [reply src/predictions/version.py:1]: Okay, reverting this.

File: src/predictions/profiles_mlcorelib/__init__.py [hunk 4]
```diff
@@ -25,6 +25,6 @@ def register_extensions(project):
 
     project.register_model_type(AuditIdStitcherModel)
 
-    # from .py_native.profiles_tutorial.tutorial import TutorialModel
+    from .py_native.profiles_tutorial.tutorial import TutorialModel
 
-    # project.register_model_type(TutorialModel)
+    project.register_model_type(TutorialModel)
```
Comment (shubhammehra4) [inline src/predictions/profiles_mlcorelib/__init__.py:30]: @dpatchigolla Please test this once, also we should skip registering for 0.6.0 IMO. Since no customer will be using it.

Comment (dpatchigolla) [reply src/predictions/profiles_mlcorelib/__init__.py:30]: Reverting this.

**Review by shubhammehra4 — APPROVED** [review]

**Review by shekhar-rudder — APPROVED** [review]
